exchanges/btcmarkets/btcmarkets_websocket.go

```diff
@@ -425,11 +425,11 @@ func orderbookChecksum(ob *orderbook.Book) uint32 {
 
 // concatOrderbookLiquidity concatenates price and amounts together for checksum processing
 func concatOrderbookLiquidity(liquidity orderbook.Levels) string {
-	var c string
+	var c strings.Builder
 	for x := range min(10, len(liquidity)) {
-		c += trim(liquidity[x].Price) + trim(liquidity[x].Amount)
+		c.WriteString(trim(liquidity[x].Price) + trim(liquidity[x].Amount))
```

Comment:
Suggestion: Might as well get rid of the concat completely in this scope. 
```suggestion
		c.WriteString(trim(liquidity[x].Price))
		c.WriteString(trim(liquidity[x].Amount))
```